akka-sample-fsm-java/README.md

```diff
@@ -1,34 +1,11 @@
 ## Finite State Machine in Actors
 
-This sample is an adaptation of [Dining Hakkers](http://www.dalnefre.com/wp/2010/08/dining-philosophers-in-humus/). It illustrates how state and behavior can be managed within an Actor with two different approaches; using `become` and using the `AbstractFSM` class.
+This sample is an adaptation of [Dining Hakkers](http://www.dalnefre.com/wp/2010/08/dining-philosophers-in-humus/). 
 
-## Dining Hakkers with Become
-
-Open [DiningHakkersOnBecome.java](src/main/java/sample/become/DiningHakkersOnBecome.java).
-
-It illustrates how current behavior can be replaced with `context.become`. Note that no `var` members are used, instead the state is encoded in the current behavior and its parameters.
-
-Start the application by typing `sbt "runMain sample.become.DiningHakkersOnBecome"` or `mvn compile exec:java -Dexec.mainClass="sample.become.DiningHakkersOnBecome"`. In the log output you can see the actions of the `Hakker` actors.
-
-Read more about `become` in [the documentation](http://doc.akka.io/docs/akka/2.6/java/actors.html#Become_Unbecome).
-
-## Dining Hakkers with FSM
-
-Open [DiningHakkersOnFsm.java](src/main/java/sample/fsm/DiningHakkersOnFsm.java).
-
-It illustrates how the states and transitions can be defined with the `akka.actor.AbstractFSM` class.
-
-Start the application by typing `sbt "runMain sample.fsm.DiningHakkersOnFsm"` or `mvn compile exec:java -Dexec.mainClass="sample.fsm.DiningHakkersOnFsm"`. In the log output you can see the actions of the `Hakker` actors.
-
-Read more about `akka.actor.FSM` in [the documentation](http://doc.akka.io/docs/akka/2.6/java/fsm.html).
-
-## Dining Hakkers with Akka Typed
-
-Open [DiningHakkersTyped.java](src/main/java/sample/typed/DinningHakkersTyped.java).
+Open [DiningHakkersTyped.scala](src/main/java/sample/DiningHakkers.java).
 
 It illustrates how the behaviors and transitions can be defined with Akka Typed.
 
-Start the application by typing `sbt "runMain sample.typed.DiningHakkersTyped"`. In the log output you can see the actions of the `Hakker` actors.
-
-Read more about Akka Typed in [the documentation](http://doc.akka.io/docs/akka/current/typed).
+Start the application by typing `sbt "runMain sample.DiningHakkers"`. In the log output you can see the actions of the `Hakker` actors.
```

Comment:
We decided to include mvn pom.xml in the java examples, and use that in run instructions. You can use this commit as template: https://github.com/akka/akka-samples/pull/141/commits/b54f1b608d38d4f1c49d6df2a023a8f4d256f5e5

Reply:
Updated